.travis.yml

```diff
@@ -75,6 +79,14 @@ script:
   - travis_retry yarn cross-env NODE_ENV=test TEST_CAFE=true testcafe electron:. ./__testcafe__/peruse.spec.ts
   - travis_retry yarn cross-env NODE_ENV=test TEST_CAFE=true testcafe electron:. ./__testcafe__/navigation.spec.ts
   - travis_retry yarn cross-env NODE_ENV=test TEST_CAFE=true testcafe electron:. ./__testcafe__/settingsMenu.spec.ts
+  - rm -rf release
+  # The reason I have done this instead of adding them as env variables to prevent windows from picking up the code signing key and trying to sign the app.
```

Comment:
The signing keys for osx need to be added as env variables. These be base 64 encode variables that could be encoded using travis-cli to and then added here to the commented out lines below. This prevents windows from picking them up and trying to sign the app.